build.gradle

```diff
@@ -85,8 +68,6 @@ task detektAllBaseline(type: io.gitlab.arturbosch.detekt.DetektCreateBaselineTas
     exclude("**/test/**/*.kt")
 }
 
-apply plugin: 'org.jetbrains.dokka'
```

Comment:
I see `kover` is not applied anywhere after this. Do we need to?

Reply:
hmmm, it wasn't applied before. In `main` it gets applied in library.gradle (https://github.com/RevenueCat/purchases-android/pull/1059/files#diff-56729f0dbf9568b5dcd9f780979f8625a04144ffd5f9e0010e579d8d44570f84R43)